src/components/search/SearchService.js

```diff
@@ -173,18 +173,19 @@ goog.require('ga_reframe_service');
             left > right ? left : right,
             right < left ? right : left
           ];
-
+          var pos95 = ol.proj.transform(position, 'EPSG:2056',
+              gaGlobalOptions.defaultEpsg);
```

Comment:
No more use of no more use of gaReframe ?  we should ask @davidoesch if it s ok.